payjoin-relay/src/main.rs

```diff
@@ -103,28 +111,50 @@ impl From<hyper::http::Error> for HandlerError {
     fn from(e: hyper::http::Error) -> Self { HandlerError::InternalServerError(e.into()) }
 }
 
+async fn post_enroll(body: Body) -> Result<Response<Body>, HandlerError> {
+    use payjoin::{base64, bitcoin};
```

Comment:
these dependencies are independent reexports of payjoin. consider depending on them directly